.github/workflows/codeqa-test.yml

```diff
@@ -43,8 +43,8 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Upgrade setuptools
-      run: pip install "setuptools >= 40.9"
+    - name: Upgrade setuptools, pip
+      run: pip install "pip>=20.0.2" "setuptools >= 44" "packaging"
```

Comment:
Why does pip have to be upgraded?
And why is packaging explicitly installed here even though it's an install dependency?
Why is the setuptools requirement here higher than in setup.cfg?

Reply:
reverting, let's see if it passes tests. I guess I was thinking of PyPy, which would require these.